### ISSUE TITLE
Discord Notifications Not showing up

### ISSUE THREAD

**Issue by LainaRaquel** [meetfranz/franz#1521]
**Describe the bug**
The bug is just that no matter what setting I enable, I don't get a notification telling me I have a message on discord. I have to remember to check it to know that I got something from someone else.

**To Reproduce**
1. Just add any discord account
2. Get a message
3. Then see you didn't get a notification on Franz and only see the notification if you click on the discord account.

**Expected behavior**
Just would like a little notification to let me know I have messages, haha.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10 
 - Browser Chrome
 - Version 5.1.0

**Comment by nelimee** [comment]
Same problem here with Franz version 5.1.0, Release 4.1.4 / linux / x64.

I have a notification sound when a message comes, but no icon change in the tray or desktop notification popping up.

**Comment by LainaRaquel** [comment]
Yes, I still haven't found a solution! But I am hoping there is one soon. :c

**Comment by nelimee** [comment]
For desktop notifications you can follow #1334 .
I did not find any solution for in-franz notification badge for the moment.

**Comment by LainaRaquel** [comment]
I have them enabled but they still don't pop up. Both ways don't work!
Hopefully it will be corrected whenever it can be D:
Or maybe discord will finally let us run multiple accounts from one
username. That would be nice. Lol.

**Comment by majesticuni** [comment]
so this started happening to me 4 days ago where i would get notificiation sounds but NOT the actual notification pop up, i was told to uninstall discord, go to localappdata and appdata folders and delete them, restart computer and reinstall, this WORKED but a day went by and i stopped getting pop-up notifications again, all settings are enabled to receive from discord so idk what is going on, i hope they fix this issue because if i have to constantly keep deleting those folders and reinstall ima be pissed, they need fix this

**Comment by bpwned** [comment]
This change fixes the issue: https://github.com/Sauceee/recipe-discord/commit/b9160ed4476af67c9389ad7819b420f626779a79#diff-e97c7f59cfcb359e6688dbab4e063c1c

**Comment by roeeyn** [comment]
Is there any advance on this issue? I still can not see discord notifications

**Comment by MrDice** [comment]
Is there any update on this issue? I'm missing the Discord notifications as well.

**Comment by Tom00214** [comment]
![Notifications](https://user-images.githubusercontent.com/76116221/102368134-871f2700-3f88-11eb-8dcc-2c1be4895712.PNG)

I right clicked on the notifications button next to the clock in windows 10, pointed to focus assist, and changed it to OFF (it was on priority only when they wouldn't appear).  This fixed it for me, not sure if it's that simple for others.

**Comment by jeremymeyers** [comment]
this is still broken?

**Comment by ZaneCode6574** [comment]
I met the same problem. I can only see the notification but no unread count on App
